experiments/GCM/heldsuarez.jl

```diff
@@ -0,0 +1,140 @@
+using LinearAlgebra
+using StaticArrays
+using Test
+using CLIMA
+using CLIMA.Atmos
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.Mesh.Filters
+using CLIMA.Mesh.Grids
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using CLIMA.VariableTemplates
+
+
+const ArrayType = CLIMA.get_array_type()
+
+Base.@kwdef struct HeldSuarezSetup{FT}
+    p_ground::FT = MSLP
+    T_initial::FT = 255
+    domain_height::FT = 30e3
+end
+
+function (setup::HeldSuarezSetup)(state, aux, coords, t) 
+    # callable to set initial conditions
+    FT = eltype(state)
+
+    r = norm(coords, 2)
+    h = r - FT(planet_radius)
```

Comment:
We should have an altitude function for this, probably in `orientation.jl`.

Reply:
Will defer this to a later PR.